recipes/sirius/v7.5/gh200/environments.yaml

```
@@ -0,0 +1,32 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.29
+      gpu: cuda
+  unify: true
+  specs:
+  - cmake
+  - cuda@12.4
+  - python@3.11
+  - py-mpi4py
+  - py-h5py
+  - py-numpy
+  - py-pybind11
+  - py-pip
+  - py-pyyaml
+  - py-scipy
+  - py-voluptuous
+  - libxc@5.2.3 ~cuda
+  - sirius@7.5 +python +tests +fortran +cuda +scalapack ^openblas
+  variants:
+  - +mpi
+  - +cuda
+  - cuda_arch=90
+  views:
+    default:
```

```suggestion
  views:
    sirius:
      link: root
```
`view:link:root` is recommended in the docs. 

Do you expect users to build SIRIUS from source themselves? If yes, we need a `develop` view too, where SIRIUS is excluded. This will likely require some tinkering on which specs do you need to add explicitly to the view so that they are available.

It is not yet documented, but the procedure to compile from source should be added to the documentation.

---

I'd skip the develop environment here. The "general public" won't build sirius, and those who do will know how to use spack upstream.

---

Given the number of root specs (and a few patches) I had to add to https://github.com/eth-cscs/alps-uenv/pull/95, I totally agree. 

However, from a private conversation with @bcumming it seemed that a `develop` view is still preferred. I'll let him comment further.

---

I have had a bit of a think about this, and how about the following:

The interfaces that you choose to provide (spack, filesystem view, modules) depend on the user community and their needs, and how the uenv maintainer intends to provide support to the users.

If you have a community that only use Spack, then you might not need views or modules for development at all.

If you have some users who will insist on "building their own" using a view or modules, then configure them in a way that will minimise your maintenance burden.

Whatever interfaces are chosen, they should be properly documented in the `/docs/<uenv-name>.md` docs.